index.html

```diff
@@ -1 +1,108 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
 
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+    Hello Camper!
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+    - The freeCodeCamp Team
+    -->
+    <nav>
+      <h3>Freecodecamp Survey Form</h3>
+      <style>
+        nav {
+          background-color: black;
+        }
+        .Love button {
+          background-color: grey;
+        }
+        .View Editor {
+          background-color: gray;
+        }
+        .Sign Up {
+          background-color: green;
+        }
+        .Login {
+          background-color: gray;
+        }
+        form {
+          text-align: center;
+        }
+        .submit button {
+          background-color: green;
+        }
+        h1 {
+          text-align: center;
+        }
+        p {
+          text-align: center;
+        }
+      </style>
+      <form action="">
```

Comment:
-Kindly apply Linters for all your Project. Linters help to check out for errors in your code so that your Code is clean.